src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -118,6 +118,7 @@ uct_cuda_copy_md_query(uct_md_h uct_md, uct_md_attr_v2_t *md_attr)
     md_attr->max_reg          = ULONG_MAX;
     md_attr->rkey_packed_size = 0;
     md_attr->reg_cost         = UCS_LINEAR_FUNC_ZERO;
+    md_attr->reg_alignment    = UCS_PGT_REG_ALIGNMENT;
```

Comment:
Maybe we could add some base_md_query() function in uct/base/uct_md.c to set default values like this, and then just call it from all MDs before tweaking specific values. That would make it easier to add or tweak fields that only differ in one MD.

Reply:
sounds good!

Reply:
done